easybuild/easyconfigs/r/R-INLA/R-INLA-25.05.04-foss-2024a.eb

```diff
@@ -0,0 +1,109 @@
+# Updated to 25.05.04
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'Bundle'
+
+name = 'R-INLA'
+version = '25.05.04'
+
+homepage = 'https://www.r-inla.org'
+description = """R-INLA is a package in R that do approximate Bayesian
+inference for Latent Gaussian Models."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('R', '4.4.2'),
+    ('R-bundle-CRAN', '2024.11'),
+    ('GSL', '2.8'),
+    ('zlib', '1.3.1'),
+    ('SuiteSparse', '7.8.2'),
+    ('METIS', '5.1.0'),
+    ('muParser', '2.3.5'),
+    ('Rmath', '%(rver)s'),
+    ('X11', '20240607'),
+    ('libtool', '2.4.7'),  # provides libltdl
+]
+
+default_easyblock = 'ConfigureMake'
+# all components are installed from the same source tarball
+default_component_specs = {
+    'source_urls': ['https://github.com/hrue/r-inla/archive/refs/tags/'],
+    'sources': [{'download_filename': 'Version_%s.tar.gz' % version, 'filename': 'R-INLA-%s.tar.gz' % version}],
```

Comment:
```suggestion
    'sources': [{'download_filename': f'Version_{version}.tar.gz', 'filename': SOURCE_TAR_GZ}],
```

Reply:
Thanks. I hope in future I remember how to do that using f-string.